test/test_pytorch_common.py

```diff
@@ -39,4 +39,4 @@ def wrapper(*args, **kwargs):
                     'Skip In CI')
 
 def flatten(x):
-    return tuple(function._iter_filter(lambda o: isinstance(o, Variable) or torch.is_tensor(o))(x))
```

Comment:
Also here `Variable` was never explicitly imported. It relies on implicit import from PyTorch side (I'm guessing `import torch.autograd.function as function`). Now that `Variable` and `Tensor` are the same, we can remove the `Variable` check.